ha_cluster_exporter.go

```diff
@@ -258,11 +258,11 @@ func main() {
 				started = true
 			}
 
-			log.Println("[INFO]: Reading ring status...")
+			log.Infoln("Reading ring status...")
```

Comment:
@stefanotorresi  the INFO here I think it might be more debug thing . see the issue we had. I'm ok if we consider this `info` -> debug for later.